app/battle.py

```diff
@@ -0,0 +1,15 @@
+from app.knight import Knight
+
+
+class Battle:
+    @classmethod
+    def two_knights_battle(cls, knights: list[Knight]):
+        for i in range(len(knights)):
+            knights[i].hp -= \
+                knights[(i + 1) % len(knights)].power - knights[i].protection
+            Battle.fix_hp_if_fell(knights[i])

```

Comment:
You need class method when you want to operate with class, but you don't use it, so it can be just function